cogs/Twitter.py

```diff
@@ -0,0 +1,693 @@
+                if tweet:
+                    logger.info(
+                        f"Processing for: @{tweet.user.screen_name} - {tweet.id_str} in {ctx.guild.name}"
+                    )
+                    await self.manage_twt(tweet, guild_id=ctx.guild.id)
+
+    @twitter.group(
+        name="configure",
+        aliases=["config"],
+        brief="Set server specific details, default_channel and role",
+        invoke_without_command=True,
+    )
+    @twitter_enabled()
+    async def configure(self, ctx):
+        await ctx.send_help(self.set)
+
+    @configure.command(
+        name="default_channel",
+        brief="Add a channel for tweets that match multiple channels to be placed into",
+    )
+    @twitter_enabled()
+    @commands.has_permissions(manage_messages=True)
+    @ack
+    async def default_channel(self, ctx, channel: discord.TextChannel = None):
+        """
+        Set a default channel for when multiple channels match a tweet.
+        Such as #group or #OT5 or #<Group_Name>
+        Leave unset to have tweet posted in all matching channels
+        Leave blank to clear default channel
+        """
+        async with self.bot.Session() as session:
+            server_settings = await db.get_twitter_settings(session, ctx.guild.id)
+            server_settings[0].default_channel = channel.id if channel else 0
+            server_settings[0].use_group_channel = bool(channel)
+
+            await session.commit()
+
+    @twitter.group(
+        name="add",
+        brief="Add sorting strategies based on channels, tags, and accounts",
+        invoke_without_command=True,
+    )
+    @twitter_enabled()
+    async def add(self, ctx):
+        await ctx.send_help(self.add)
+
+    @add.command(
+        name="hashtag",
+        brief="Add a channel for a member's pictures or videos to be posted in",
+    )
+    @twitter_enabled()
+    @commands.has_permissions(manage_messages=True)
+    @ack
+    async def add_hashtag(
+        self, ctx, channel: discord.TextChannel, hashtag: commands.clean_content
+    ):
+        """
+        Associates a hashtag with a specific channel, more than one hashtag can be associated with any channel
+
+        Example usage:
+        `{prefix}twitter add hashtag <channel> <hashtag>`
+        `{prefix}twitter add hashtag #Irene 아이린`
+        """
+        # in case they enter it with the hashtag character
+        tag_conditioned = hashtag.split("#")[-1]
+        # check if hashtag already exists
+        async with self.bot.Session() as session:
+            twitter_sorting = await db.get_twitter_sorting(
+                session, hashtag=tag_conditioned, guild_id=ctx.guild.id
+            )
+            if twitter_sorting:
+                # twitter_sorting[0].channel = channel.id
+                await db.delete_twitter_sorting(session, tag_conditioned, ctx.guild.id)
+                logger.info(f"{ctx.guild.name} updated {tag_conditioned} to #{channel}")
```

Comment:
```suggestion
                logger.debug(f"{ctx.guild.name} updated {tag_conditioned} to #{channel}")
```
Most of these would be pretty spammy. Not sure whether I'd want to see them in my info logs